src/caselawclient/models/documents/__init__.py

```diff
@@ -340,22 +340,34 @@ def force_enrich(self) -> None:
             now.isoformat(),
         )
 
+        if not self.can_enrich:
+            msg = f"{self.uri} has no docx"
```

Comment:
This message is incorrect:

```suggestion
            msg = f"{self.uri} has no body content"
```